R/update-regional.R

```diff
@@ -23,38 +23,43 @@ update_regional <- function(location, excludes, includes, force, max_execution_t
   # Update delays -----------------------------------------------------------
   if (!is.list(location$generation_time)) {
     futile.logger::flog.trace("loading generation_time.rds")
-    location$generation_time <- readRDS(here::here("data", "generation_time.rds"))
+    location$generation_time <- readRDS(here::here("data", "generation_time.rds")) #suggest moving into list def
```

Comment:
I can see that - currently you can override by providing an alternative pre-loaded list into the generation_time param on the dataset but it would be nicer if it could be loaded only at execution time.

Reply:
agree.